Changes to structure

RomanSavran opened the pull request:
- All attributes added to ontology and mapped to domains
- Context and vocabulary files autogenerated
- Ontology file changed due to latest requerements
- Documentation file changed

RomanSavran committed:
Data model structure initial

New version of Data model

RomanSavran committed:
Update version (fixed atributes case)

RomanSavran committed:
Refactoring of class names and contexts

Class rename (pot:Object -> pot:Thing), add nested attributes to context

RomanSavran committed:
Add nested attributes

Added nested attributes to context and vocabulary files

soderluk commented on vocabularies/person.jsonld:
```diff
@@ -0,0 +1,47 @@
+                "dli:description": "data",
+                "dli:required": true,
+                "dli:valueType": "xsd:object"
+            }
```
There should probably be some more information about a person. E.g. optional attributes such as date of birth etc. other useful information that a "person" can have.

RomanSavran replied:
Redirected this to Sebastian - he is providing attributes list e.t.c.

RomanSavran replied:
Will merge this and work with Sebastian on attributes on Person and other classes in next commits

soderluk commented on vocabularies/link.jsonld:
```diff
@@ -1,32 +1,47 @@
+                "dli:description": "data",
+                "dli:required": true,
+                "dli:valueType": "xsd:object"
+            }
```
Please revert this file, as links do only have a type. No name or data attribute at the moment.

soderluk commented on README.md:
```diff
@@ -1,182 +1,63 @@
+
+The decision to allow multiple domains and ranges was purely pragmatic. While the computational properties of systems with a single domain and range are easier to understand, in practice, this forces the creation of a lot of artifical classes, which are there purely to act as the domain/range of some properties.
+
+All classes and properties can be investigated in [Data Model structure documentation](https://romansavran.github.io/PoT/).
```
Please point to the standards repository instead.

soderluk commented on README.md:
```diff
@@ -1,182 +1,63 @@
+All JSON-LD source files are available under [GitHub Repository](https://github.com/PlatformOfTrust/standards).
+
+## Links and additional information
+All classes and properties can be investigated in [Data Model structure documentation](https://romansavran.github.io/PoT/).
```
Please point to the standards repository.

soderluk reviewed (changes requested):
You have removed all the contexts for links, which we need in the system. Please restore ALL link-*.jsonld contexts, and fix the vocabularies for them.
Also, you removed one of the most important context file, e.g. identity-app.jsonld, please restore the identity-app.jsonld context.

soderluk reviewed (commented):
The context file identity-data_product.jsonld is removed, please restore it.

soderluk reviewed (commented):
The identity-link.jsonld shouldn't exist. A link is not an identity, it's a link.
Links do not have a name or data, only the type

soderluk commented:
Could you at the same time update (replace) the base URL from `https://platformoftrust.github.io/standards/` to `https://standards.oftrust.net/`
Should be pretty straight forward to find and replace the previous string with the latter.

RomanSavran committed:
Fixes according to pull request Changes to structure #3 feedback

- Add deleted files required by application
- Replace https://platformoftrust.github.io/standards/ to https://standards.oftrust.net/

RomanSavran committed:
Update pot.jsonld

New Class structure

RomanSavran committed:
Numerous subclasses added

- Building types
- Appartment types
- Room types
- Building systems and elements
- Electric device types

soderluk commented:
@RomanSavran: Please merge master into your branch, and resolve the conflicts.

RomanSavran committed:
Merge remote-tracking branch 'upstream/master'

RomanSavran committed:
Add DLI classes used in PoT

- Generated contexts and vocabularies for DLI classes used in PoT

RomanSavran commented:
@soderluk I completed pot ontology part, but it seems like DLI ontology https://digitalliving.github.io/standards/ontologies/dli.jsonld is outdated (it is missing some classes like Building) even so this classes should be there instead of pot according to latest input I have

Is it possible to provide me access to https://digitalliving.github.io/standards/ so I can add missing classes or how would it better solve this?

soderluk commented:
@RomanSavran Did you get the invite?

RomanSavran commented:
@soderluk Got it, thank you
